aten/src/ATen/native/Convolution.cpp

```diff
@@ -394,7 +394,7 @@ auto ConvParams::cudnn_use_channels_last(
   }
   long cudnn_version = detail::getCUDAHooks().versionCuDNN();
   // old cudnn version has sparse group convolution support for NHWC layout
-  return (groups == 1 || cudnn_version >= 7603) &&
+  return (cudnn_version >= 7603) &&
```

Comment:
Maybe we could update the comments above, as it is not relevant with the `group==1` removed.